app/container/volumes/template.hbs

```diff
@@ -20,7 +20,7 @@
         </td>
         <td data-title={{t 'containersPage.containerPage.volumesTab.table.body.shared'}}>
           <ul class="list-unstyled">
-          {{#each mount.volume.sharedWith as |container|}}
+          {{#each mount.sharedContainers as |container|}}
```

Comment:
This doesn't appear to have any way to hide the container you're looking at from the list of ones the mount is shared with?